core-web/libs/dotcms-scss/shared/_common.scss

```diff
@@ -12,6 +12,10 @@ $field-height-lg: 3rem; // 48px
 
 $field-border-size: 1.5px;
 
+$chip-height-sm: 1.5rem; // 24px
+$chip-height-md: 2.5rem; // 40px
```

Comment:
Do this inside the chips if you actually need it.

Reply:
Moved.